src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.ts

```diff
@@ -95,7 +104,7 @@ export const createDashboard = async (
   // Build and return the dashboard container.
   // --------------------------------------------------------------------------------------
   const initialComponentState: DashboardPublicState = {
-    lastSavedInput: savedObjectResult?.dashboardInput ?? {
+    lastSavedInput: omit(savedObjectResult?.dashboardInput, 'controlGroupInput') ?? {
```

Comment:
The dashboard container no longer needs to remember/keep track of the `controlGroupInput`, because the control group now handles all of the unsaved changes logic internally; however, since control groups are still saved as **part** of a dashboard SO, the SO still needs to contain the `controlGroupInput`.